pages/_app.tsx

```diff
@@ -6,11 +6,12 @@ import "typeface-inter";
 import { useEffect } from "react";
 
 import { LayoutRoot } from "~/components/layout/layout-root";
-import { Script } from "~/components/script";
```

Comment:
Please remove the custom `Script` component as well, thanks!

Reply:
already did!